index.js

```diff
@@ -1015,3 +1020,8 @@ function makeError (message, code) {
   err.code = code
   return err
 }
+
+function warn (message) {
+  if (console === undefined) return
+  console.warn(message)
```

Comment:
I'm wondering in which case `console` can be unavailable?

Reply:
I can't think of one.